### ISSUE TITLE
Register bitgroom filter with HDF Group

### ISSUE THREAD

**Issue by edwardhartnett** [ccr/ccr#136]
Has the bitgroom filter been registered with the HDF Group? If not, it should be. If so, we should mention that in the docs.

Can the bitgroom filter be obtained through the HDF filter package? If not, probably we should add it there too.

**Comment by czender** [comment]
No I just made up the filter ID, and yes, we should register it with THG. When do you want to release 1.1.0? Can we please not release 1.1.0 until the week before AMS? That will allow sufficient time for me to document what you have asked for and register with THG. There is also the issue that there is already a Zstandard filter ID, which I just copied into our filter, even though our filter shares no source code with the registered one. I wrote a new filter for CCR because the old one was a bit out-of-date and did not exploit newer versions of Zstandard, and had deprecated function calls. I think it's OK to share the ID because the CCR  filter is a superset of the other (it accepts more compressions levels, and handles out-of-range CLs more gracefully). What do you think about CCR using the existing Zstandard ID @edwardhartnett ?

**Comment by edwardhartnett** [comment]
I have made a separate issue for the Zstandard questions (#137).

We can hold the release until you get the bitgroom filter registered. Presumably the HDF group will assign an ID that should be used instead of what we've got now...

**Comment by edwardhartnett** [comment]
OK, looks like the way to register this is to notify Barbara at the HDF Group. We just need to send an email to 
help@hdfgroup.org. Are we ready to send that email?

**Comment by edwardhartnett** [comment]
OK, Charlie sent the email:

> Hello HDF Group,
> 
> Please assign the BitGroom filter an HDF5 Filter ID.
> The BitGroom quantization algorithm is documented in
> 
> Zender, C. S. (2016), Bit Grooming: Statistically accurate
> precision-preserving quantization with compression, evaluated in the
> netCDF Operators (NCO, v4.4.8+), Geosci. Model Dev., 9, 3199-3211,
> doi:10.5194/gmd-9-3199-2016.
> 
> The filter is documented and maintained in the
> Community Codec Repository (https://github.com/ccr/ccr).
> 
> Thank you,
> Charlie

**Comment by edwardhartnett** [comment]
Looks like we're good to go!

> Hi Charlie,
> 
> I entered filter id 32022 for BitGroom.
> 
> See the table on this page:
>   https://portal.hdfgroup.org/display/support/Registered+Filter+Plugins
> 
> and the entry here:
>   https://portal.hdfgroup.org/display/support/Filters#Filters-32022
> 
> If you want me to change any words, just let me know!
> 
> -Barbara

**Comment by czender** [comment]
This will be done once #144 is merged.

**Comment by czender** [comment]
Fixed via #144
